source/js/buyers-guide/homepage-c-slider.js

```diff
@@ -62,12 +62,17 @@ export default {
 
       let n = visible.length;
 
-      // shortcut this scroll update if there are no products
+      // Shortcut this scroll update if there are no products
       if (n===0) { return; }
 
-      let averageCreepiness = visible.reduce( (tally, v) => tally + parseFloat(v.dataset.creepiness)/n, 0);
+      // Reduce the creepiness, treating any product without votes
+      // as a "neutral" product with creepiness = 50
+      let averageCreepiness = visible.reduce( (tally, v) => {
+        let value = parseFloat(v.dataset.creepiness) || 50;
```

Comment:
without this check the resultant creepiness for products without a value (yet) would end up being "undefined", which is not one of the five possible creepiness labels =)